_data/nav.yaml

```diff
@@ -10,13 +10,5 @@
 - title: Coalition
   url: /coalition
 
-- title: Jobs
-  url: https://impactlabs.notion.site/Current-Openings-at-Impact-Labs-3c6a003cb601470298ab9454949bea75
-  target: true
-
-- title: Team
-  url: /team
-
-- title: Blog
```

Comment:
Why did we remove the blog?

Reply:
I was thinking something we could replace this with is "Press" and have a dedicated section to any media that we've seen about Impact Labs online

Reply:
Yes I think that would be great! Let's still remove it here since I think it looks a bit bad that it's so out of date, and then in the future we can have a separate ticket to add a whole Press page, maybe after the Substack is set up and we start publishing interviews